scss/samples/_variables.scss

```diff
@@ -0,0 +1,40 @@
+// Sample Variable Documentation
+// =============================
+
+
+// Private
+// -------
+/// Private variables will display unless you set `access: ['public']`
+/// in your [SassDoc config](http://sassdoc.com/configuration/).
+/// @group samples
+/// @access private
+/// @type *
+$private: 'for internal use only';
+
+
+// Basic Value
+// -----------
+/// The basics for documenting your average non-map variable.
+/// @group samples
+/// @type *
```

Comment:
`/// @type *` this confused me at first. Does it just mean it is a global variable? If it weren't, what would type be?